src/Paket.Core/InstallProcess.fs

```diff
@@ -254,37 +236,37 @@ let private applyBindingRedirects isFirstGroup createNewBindingFiles redirects c
                 librariesForPackage
                 |> Seq.choose(fun (library,redirects,profile) ->
                     try
-                        let assembly = LoadAssembliesSafe.reflectionOnlyLoadFrom library
-                        Some (assembly, BindingRedirects.getPublicKeyToken assembly, assembly.GetReferencedAssemblies(), redirects, profile)
+                        let assembly = Mono.Cecil.AssemblyDefinition.ReadAssembly(library)
+                        Some (assembly, BindingRedirects.getPublicKeyToken assembly, assembly.MainModule.AssemblyReferences, redirects, profile)
```

Comment:
Was there a reason for not using Mono.Cecil until now? Everything still seems to work and the code looks "safer" now. But maybe there was a reason to use `ReflectionOnlyLoadFrom`?